jsEngine/settings/Settings.ts

```diff
@@ -1,14 +1,22 @@
 import type JsEnginePlugin from 'jsEngine/main';
 import { StartupScriptsModal } from 'jsEngine/settings/StartupScriptModal';
-import type { App } from 'obsidian';
-import { PluginSettingTab, Setting } from 'obsidian';
+import type { App , TFile } from 'obsidian';
+import { normalizePath, PluginSettingTab, Setting } from 'obsidian';
+
+declare module 'obsidian' {
```

Comment:
Please add this to a separate `obsidian-ex.d.ts` file.

Reply:
should I add a new file inside `jsEngine/settings/` or extend the existing `jsEngine/obsidian-ex.d.ts`?